contrib/cirrus/win-podman-machine-test.ps1

```diff
@@ -33,5 +33,8 @@ Run-Command ".\bin\windows\podman.exe --version"
 New-Item -ItemType "directory" -Path "$env:AppData\containers"
 Copy-Item -Path pkg\machine\ocipull\policy.json -Destination "$env:AppData\containers"
 
+Write-Host "`nRunning podman-machine unit tests"
+Run-Command ".\winmake localunit"
+
```

Comment:
This should not done like this.

Merging both tests will make it hard to figure out what failed. But most importantly this breaks all our CI assumptions that we have in our skip tests based on sources. machine tests will only run on machine changes, unit tests might be elsewhere, I do see that this is not the case right now but the next person will overlook this when they enable windows unit tests outside of pkg/machine.

This should really be its own top level task I would think like the existing unit test. That also means we do run them twice for WSL and hyperV, I guess right now they are fast enough so that this doesn't matter much but still.

Reply:
You are right. I have added a new top level cirrus task `unit podman windows rootless host sqlite`.